pkg/build/controller/build/build_controller.go

```diff
@@ -1210,6 +1216,18 @@ func (bc *BuildController) createBuildPod(build *buildv1.Build) (*buildUpdate, e
 				return update, err
 			}
 		}
+		if bc.globalProxySet {
+			hasProxyCAMap, err := bc.findOwnedConfigMap(existingPod, build.Namespace, buildutil.GetBuildProxyCAConfigMapName(build))
+			if err != nil {
+				return update, fmt.Errorf("could not find proxy certificate authority for build: %v", err)
+			}
+			if !hasProxyCAMap {
+				update, err = bc.createBuildProxyCAConfigMap(build, existingPod, update)
+				if err != nil {
+					return update, err
+				}
+			}
+		}
```

Comment:
Ditto - this should be commented out until the new trusted CA controller lands.

Reply:
part of next push